openapi/components/schemas/RiskScoreSimulation.yaml

```
@@ -0,0 +1,57 @@
+type: object
+description: Risk score simulation.
+required:
+  - rules
+properties:
+  id:
+    description: ID of the simulation.
+    type: string
+    readOnly: true
+    maxLength: 50
+    example: rss_0YVCNQ2C1AD1RA3HXKP492GNZB
+  rules:
+    description: Risk score rules.
+    $ref: ./RiskScoreRules.yaml
+  status:
+    description: Status of the simulation.
+    type: string
+    readOnly: true
+    enum:
+      - pending
+      - processing
+      - completed
+      - failed
+  summary:
+    description: Summary of the simulation.
+    readOnly: true
+    type:
+      - array
+      - null
```

```suggestion
      - 'null'
```

String interpolation in YAML: null in YAML versus "null" type (types are strings).